README.md

```diff
@@ -5,21 +5,26 @@ Kueue is a set of APIs and controller for [job](docs/concepts/workload.md)
 a job should be [admitted](docs/concepts#admission) to start (as in pods can be
 created) and when it should stop (as in active pods should be deleted).
 
+## Why use Kueue
+
+Kueue is a lean controller that you can install on top of a vanilla Kubernetes
+cluster. Kue does not replace any existing Kubernetes components. Kueue is
```

Comment:
```suggestion
cluster. Kueue does not replace any existing Kubernetes components. Kueue is
```

Reply:
Ah, how did that happen?
Although not a bad name (it's basically pronounced the same).